### PR TITLE
Feature/person page

### PR DESCRIPTION
**Pull request by nikitius20**

### CODE REVIEW

**Comment by skss-1** [comment]
Please, update your branch by latest develop

File: src/components/CreditsScroll.vue [hunk 1]
```diff
@@ -0,0 +1,87 @@
+<template>
+  <div 
+    class="credits-scroll my-3 p-2"
+    tabindex="0"
```
Comment (skss-1) [inline src/components/CreditsScroll.vue]: Should not be the 'tabindex' on parent element. Only cards which can be clickable.
Scroll on this section will work when focus on cards

File: src/components/ItemsList.vue [hunk 2]
```diff
@@ -1,5 +1,8 @@
 <template>
-  <div class="items-list container p-0">
+  <div
+    class="items-list container p-0"
+    tabindex="0"
```
Comment (skss-1) [inline src/components/ItemsList.vue]: Should not be a 'tabindex' on parent element.

File: src/components/MovieCard.vue [hunk 3]
```diff
@@ -1,6 +1,11 @@
 <template>
-  <div class="col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0">
-    <div class="card border-light">
+  <div class="movie-card col-lg-2 col-md-2 col-sm-3 col-5 m-3 p-0"> 
+    <div
+      class="card"
+      tabindex="0"
+      @click="goToMoviePage"
+      @keydown.enter.esc="goToMoviePage"
```
Comment (skss-1) [inline src/components/MovieCard.vue]: `esc` key is not used for `open` action. Please, remove the 'esc' event handling

**Review by skss-1 — CHANGES_REQUESTED** [review]

File: src/components/CreditsScroll.vue [hunk 4]
```diff
@@ -1,9 +1,15 @@
...
+  >
     <div
       v-for="actor in credits"
       :key="actor.id"
       class="card actor-card d-flex"
+      tabindex="0"
+      @click="goToPersonPage(actor.id)"
+      @keydown.enter.esc="goToPersonPage(actor.id)"
```
Comment (skss-1) [inline src/components/CreditsScroll.vue]: `esc` event handling is not needed here

**Review by OleksandrBielik — APPROVED** [review]

**Review by skss-1 — APPROVED** [review]

**Pull request merged**